classes/MatomoReportingService.php

```
@@ -0,0 +1,387 @@
+            'format' => 'json',
+            'token_auth' => $this->authToken,
+            'urls' => $bulkUrls,
+        ]);
+    }
+
+    /**
+     * Clears only cache entries generated by this service.
+     */
+    public function clearCache(): void
+    {
+        $keys = Cache::get(static::CACHE_INDEX_KEY, []);
```

PHPStan (2.1.54)

[error] Unsafe access to private constant Winter\Matomo\Classes\MatomoReportingService::CACHE_INDEX_KEY through static::.


(staticClassAccess.privateConstant)